### PR TITLE
Updated Polish translation

### PR DESCRIPTION
**Pull request by inxaneninja**
Updated missing translations

### CODE REVIEW

File: core/assets/bundles/bundle_pl.properties [hunk 1]
```diff
@@ -159,15 +159,15 @@ save.rename = Zmień nazwę
...
+editmessage = Edytuj wiadomość
 save.corrupted = [accent]Zapis gry jest uszkodzony lub nieprawidłowy! Jeżeli aktualizowałeś grę, najprawdopodobniej jest to zmiana w formacie zapisu i [scarlet]nie jest[] to błąd.
 empty = <pusto>
 on = Włączone
 off = Wyłączone
 save.autosave = Autozapis: {0}
 save.map = Mapa: {0}
 save.wave = Fala {0}
-save.mode = Gamemode: {0}
+save.mode = Tryb: {0}
```
Comment (rafalkotowicz) [inline core/assets/bundles/bundle_pl.properties]: Maybe "Tryb gry"?

Comment (azureblue) [reply core/assets/bundles/bundle_pl.properties]: All those change already exists in #765...

**Comment by azureblue** [comment]
All those changes already exists in #765...

**Comment by Anuken** [comment]
Closing, as this appears to be unnecessary now.

**Comment by azureblue** [comment]
@Anuken I see you started merging translation changes. Is there going to be a release soon? :wink: If it is, I'll look if there are any missing translations asap.

**Comment by Anuken** [comment]
Not planning on releasing *too* soon; I'm only merging because there's a very large backlog of translations, and I won't have time to merge them all if I don't start now.

**Pull request closed without merging**